codegen/example-yaml/main.go

```diff
@@ -115,11 +115,20 @@ func main() {
 func createExampleSeed(config *kubermaticv1.KubermaticConfiguration) *kubermaticv1.Seed {
 	imageList := kubermaticv1.ImageList{}
 	operatingSystemProfileList := kubermaticv1.OperatingSystemProfileList{}
-
+	kubevirtHTTPSource := kubermaticv1.HTTP{
+		OperatingSystems: map[providerconfig.OperatingSystem]kubermaticv1.OSVersions{},
+		ImageCloning: &kubermaticv1.ImageCloning{
+			Enable:       false,
+			StorageClass: "",
+		},
+	}
 	for _, operatingSystem := range providerconfig.AllOperatingSystems {
 		imageList[operatingSystem] = ""
 		operatingSystemProfileList[operatingSystem] = ""
 	}
+	for supportedOS := range kubermaticv1.SupportedKubeVirtOS {
+		kubevirtHTTPSource.OperatingSystems[supportedOS] = nil
```

Comment:
It might help to have not `nil` but an example map so the `zz_generated_seed.xx.yaml` would contain a full example.